src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -5,7 +5,8 @@ export default {
   create(context) {
     return {
       MemberExpression: function(node) {
-        if (node.object.object &&
+        if (!node.computed &&
```

Comment:
What is the potential impact on schema validation for all APIS. Presumably this would mean we'd skip everything that's dynamically referenced. Are reviewers ok with that vs the flipside of more false positives?

/cc @wagnerand